docs/en/enterprise-edition/policy-reference/sast-policies/javascript-policies/sast-policy-263.adoc

```diff
@@ -0,0 +1,43 @@
+== Risky usage of malicious Polyfill.io library
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|=== 
+|Prisma Cloud Policy ID 
+| 5e39ff6f-a442-462f-93e3-50a551b138f7
+
+|Checkov ID 
+|CKV3_SAST_263
+
+|Severity
+|MEDIUM
+
+|Subtype
+|Build
+
+|Language
+|JavaScript
+
+|=== 
+
+=== Description
+
+This policy identifies the usage of the Polyfill.io library in JavaScript applications. Polyfill.io has been known to serve malware, and the `polyfill-library` should not be used. It is recommended to avoid using this library to prevent potential security risks.
```

Comment:
This policy checks whether the **Polyfill.io** library is used in JavaScript applications. Polyfill.io has a history of serving malware; therefore, using the polyfill-library is strongly discouraged.

@tsmithv11 moved the remediation